hack/script_nfs.bash

```diff
@@ -37,6 +44,5 @@ helm upgrade nfs-subdir-external-provisioner \
      --set nfs.path=/srv/ocpstorage \
      --wait --atomic --install
 
-oc adm policy add-scc-to-user hostmount-anyuid \
-   system:serviceaccount:aicoe-meteor:nfs-subdir-external-provisioner
```

Comment:
Regarding that, I think we should keep aicoe-meteor hardcoded for now : unless I'm mistaken, it's hardcoded in the kustomize files (
```console
$ grep -R aicoe-meteor
config/base/kustomization.yaml:namespace: aicoe-meteor
config/default/kustomization.yaml:namespace: aicoe-meteor
config/olm/operatorgroup.yaml:  namespace: aicoe-meteor
config/olm/operatorgroup.yaml:    - aicoe-meteor
grep: bin/manager: binary file matches
pipelines/base/kustomization.yaml:namespace: aicoe-meteor
README.md:The following commands will build the image, push it to the container registry, and deploy the operator to the `aicoe-meteor` namespace of your currently configured cluster (as per your `$KUBECONFIG`):
main.go:		LeaderElectionNamespace: "aicoe-meteor",
```
)

Reply:
But this script is about the NFS provisioner, right?  And that can be in a different namespace. In fact, I believe it's preferrable to have it elsewhere... in my current test cluster I have a dedicated namespace for it, but I've also used `default` or some other.

In a test/dev environment, `make undeploy` will delete the `aicoe-meteor` namespace; if the NFS provisioner is there, it will get deleted too.

Reply:
Ok, I was under the impression that there was some sort of necessity to have it in the same ns, but if not, that's fine :).
Although, in that case, maybe we want to explicit create a dedicated ns for the nfs-provisionner (or put in in `default`), rather than leave it in the current namespace. That would make the install more predictable and test env would be more alike.
wdyt ?

Reply:
Right, that makes sense. I will add a dedicated namespace